src/amo/components/AddonReviewListItem/styles.scss

```diff
@@ -14,6 +14,10 @@
   .ErrorList {
     margin-top: 6px;
   }
+
+  .TooltipMenu {
+    width: 220px;
```

Comment:
I wonder why the width is so much in the first place... this seems good but setting a width on it might mess up other locales. How does this look in German, for instance?

I wonder if `width: auto` as the default here is messing things up and a `width: 100%;` would fix it?

Reply:
Width 100% on TooltipMenu component is making things even worth:

<img width="1173" alt="screen shot 2018-04-05 at 12 56 56 pm" src="https://user-images.githubusercontent.com/31850821/38380137-dcfb8266-38d0-11e8-88d5-80005e5b6730.png">

I have changed width to `fit-content` and it looks good in both German and English now:

<img width="399" alt="screen shot 2018-04-05 at 12 52 54 pm" src="https://user-images.githubusercontent.com/31850821/38380205-09327a4c-38d1-11e8-8212-a0a28f04abc0.png">

<img width="498" alt="screen shot 2018-04-05 at 12 51 50 pm" src="https://user-images.githubusercontent.com/31850821/38380200-06e9058a-38d1-11e8-98cd-b968b9414b05.png">

I am not sure if it's ok to use `fit-content`, it's supported in (according to [this website](https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content)):

Chrome | Edge | Firefox | Internet Explorer | Opera | Safari
-- | -- | -- | -- | -- | --
 29 | 16 | 51 | No | 44 | 10.1
 
This message in English is this weird because it's width is set to be 320px [here](https://github.com/mozilla/addons-frontend/blob/8e471ffcea03dc86ec152b4613a856d1b451e838/src/ui/components/TooltipMenu/styles.scss#L13). In German it looks perfectly fine with width 320px.